Sim/SimDelphesInterface/options/PythiaDelphes_config.py

```diff
@@ -40,6 +40,12 @@ def apply_paths(obj, names):
   for attr, name in names.iteritems():
     getattr(obj.DataOutputs, attr).Path = name
 
+
+import argparse
+parser = argparse.ArgumentParser()
+parser.add_argument('--delphescard', type=str, default='', help='specify an input delphes card')
```

Comment:
if the default is an invalid value - and an empty string is - one should say default=None. I am nitpicking though.